sqlglot/tokens.py

```diff
@@ -221,6 +221,7 @@ class TokenType(AutoName):
     AUTO_INCREMENT = auto()
     BEGIN = auto()
     BETWEEN = auto()
+    BULK_COLLECT_INTO = auto()
```

Comment:
Let's not add a token, instead do match texts, we generally don't add tokens unless we have to.